packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -317,6 +328,12 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
     }
 
     async registerToolbarItems(toolbarRegistry: TabBarToolbarRegistry): Promise<void> {
+        toolbarRegistry.registerItem({
+            id: FileNavigatorCommands.REFRESH_NAVIGATOR.id,
+            command: FileNavigatorCommands.REFRESH_NAVIGATOR.id,
+            tooltip: 'Refresh',
```

Comment:
Can we use `Refresh Explorer` as a tooltip to align with VSCode?

Reply:
Thanks, I've updated it now